pageserver/src/http/openapi_spec.yml

```diff
@@ -1379,6 +1379,47 @@ paths:
               schema:
                 $ref: "#/components/schemas/ServiceUnavailableError"
 
+  /v1/utilization:
+    get:
+      description: |
+        Returns the pageservers current utilization and fitness score for new tenants.
+
+      responses:
+        "200":
+            description: Pageserver utilization and fitness score
+            content:
+              application/json:
+                schema:
```

Comment:
NIT: for clarity, it could be better to separate this into a new schema like `UtilizationInfo` and just reference here, see `TenantInfo` as example

Reply:
Agreed, will move.

Reply:
Moved in https://github.com/neondatabase/neon/pull/6831/commits/3214df737d032aca1ba4258dd494ee851cf43e72